HerPublicWebsite.ManagementShell/CommandHandler.cs

```diff
@@ -25,12 +25,32 @@ public void GenerateReferrals(string[] args)
         if (!int.TryParse(args[0], out var referralCount))
             outputProvider.Output("Invalid number of referrals. Usage: 'GenerateReferrals <count>'.");
 
+        const string environmentKey = "ASPNETCORE_ENVIRONMENT";
```

Comment:
This is kind of reusing something else for a purpose it wasn't designed for, but think that's fine here. Not particularly worried.